### DISCUSSION TITLE
Too few arguments to function App\Http\Controllers\Agent\helpdesk\MailController::get_data(), 0 passed in

### DISCUSSION THREAD

**Discussion by felipediazrosas** [ladybirdweb/faveo-helpdesk#8135] [General]
Too few arguments to function App\Http\Controllers\Agent\helpdesk\MailController::get_data(), 0 passed in

**Comment by RafficMohammed** [comment]
@felipediazrosas Can u show me arguments where u r passing in get_data method()
